<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DapUiWidgetChainBalance</class>
 <widget class="QWidget" name="DapUiWidgetChainBalance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="a_balance">
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size:30px;
	color: red;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="a_simpl">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size:15px;

}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tockIcon">
       <property name="text">
        <string>icon</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="a_gold">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size:15px;

}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tockIcon_2">
       <property name="text">
        <string>icon</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="a_another">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size:15px;

}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tockIcon_3">
       <property name="text">
        <string>icon</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
